composer.json

```diff
@@ -27,7 +27,7 @@
     "require": {
         "php": "^7.0|^8.0",
         "psr/container": "^1.0|^2.0",
-        "phpstan/phpstan": "^1.0"
+        "phpstan/phpstan": "^2.0"
```

Comment:
maybe ^1.0|^2.0 is also possible.

Reply:
thx, tested. This also still works with v1, so I'll readd that.